### PR TITLE
Qaft 673 banking flow

### PR DESCRIPTION
**Pull request by murad755**

### CODE REVIEW

File: cypress/support/Helpers/common/messages.js [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
 
-export const checkErrorMessage = (message) => {
-    cy.get(messageSelectors.error).should('contain', message)
+export const checkMessage = (selector, message) => {
+    cy.get(selector).should('contain', message)
+}
+
+export const agreementError = (message) => {
+    cy.get(messageSelectors.agreementError).should('contain', message)
+}
```
Comment (Praisy02) [inline cypress/support/Helpers/common/messages.js:27]: hey @murad755 : can you use checkMessage() method for validating agreement error ?

File: cypress/integration/nebula/bankingFlow.spec.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        checkButtonIsActive('Finish')
+        clickButtonByValue('Finish')
+
+       signUpPage
+           .selectItemFromNavbar('Invoices')
+
+        checkNotification('Step completed')
+        checkNotification('Thank you for confirming your bank account.')
+
+        bankingFlowPage.checkCard('Add your bank', false)
```
Comment (Praisy02) [inline cypress/integration/nebula/bankingFlow.spec.js:85]: can you handle the case if action card doesn;t exist ?

**Review by Praisy02 — CHANGES_REQUESTED** [review]

File: cypress/support/Helpers/common/messages.js (hunk 1 above)
Comment (murad755) [reply cypress/support/Helpers/common/messages.js]: done

File: cypress/integration/nebula/bankingFlow.spec.js (hunk 2 above)
Comment (murad755) [reply cypress/integration/nebula/bankingFlow.spec.js]: done

**Pull request merged**